docs/faq.rst

```diff
@@ -287,7 +287,7 @@ How do I perform operations in a request handler after sending the response?
 
 Middlewares can be written to handle post-response operations, but
 they run after every request. You can explicitly send the response by
-calling :meth:`aiohttp.web.Response.write_eof`, which starts sending
+calling :meth:`aiohttp.web.StreamResponse.write_eof`, which starts sending
```

Comment:
This section documents `aiohttp.web.Response`. It would be weird to suddenly point at something else.

Reply:
Okay, `aiohttp.web.Response.write_eof`  # undocumented